codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -209,22 +287,37 @@ public void incrementWindowSize(Http2Stream stream, int delta) throws Http2Excep
         assert ctx == null || ctx.executor().inEventLoop();
         if (stream.id() == CONNECTION_STREAM_ID) {
             // Update the connection window
-            connectionState().incrementStreamWindow(delta);
+            final AbstractState connectionState = connectionState();
+            connectionState.incrementStreamWindow(delta);
+
+            // Check if we need to notify listener of writability changes
+            if (listener != null && connectionState.isWritabilityChanged()) {
+                connectionState.resetWritabilityChanged();
+                // All streams with windows > 0 have changed writability
+                connection.forEachActiveStream(new Http2StreamVisitor() {
```

Comment:
Yes. I could do this in a few places, however they cannot be static because `state(..)` uses a property key which does not support static use cases.  So I could have up to 4 extra member variables for iterators. Let me try this.